SOURCES/xapi-1.214.1-create-plugged-vif-and-vbd-for-suspended-vm.XCP-ng.patch

```diff
@@ -0,0 +1,401 @@
+This patch does 2 things:
+* add device and currently_attached to the VBD.create method to allow the creation of an already plugged VBD. the new fields can only be used if the VM is suspened (else it throws). Default behaviour is kept.
+* add currently_attached to the VBD.create method to allow the creation of an already plugged VIF. the new fields can only be used if the VM is suspened (else it throws). Default behaviour is kept.
```

Comment:
Is it `VIF.create` instead of `VBD.create` here? (because we are talking about VIF later in the sentence

Reply:
IDK what you're talkin about. *Whistle innoncently*
:p